src/directConnectManager.test.ts

```diff
@@ -38,10 +36,6 @@ const PLAIN_LOCAL_KAFKA_SR_SPEC: ConnectionSpec = {
 describe("DirectConnectionManager behavior", () => {
   let sandbox: sinon.SinonSandbox;
 
-  let onDidChangeConfigurationStub: sinon.SinonStub;
-  let getConfigurationStub: sinon.SinonStub;
-  let setContextValueStub: sinon.SinonStub;
```

Comment:
Unused after workspace config tests moved in https://github.com/confluentinc/vscode/pull/725